### PR TITLE
Lesson10

### PR DESCRIPTION
**Pull request by vtx123**

### CODE REVIEW

File: Lesson10/Lesson10_1.cpp [hunk 1]
```diff
@@ -0,0 +1,41 @@
+// Lesson 10 1
+#include <iostream>
+
+void swap(int& a, int& b) {
+    int tmp = 0;
```
Comment (helen-black) [inline Lesson10/Lesson10_1.cpp]: int tmpp = a;

File: Lesson10/Lesson10_2.cpp [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+    for(int i = 0; i < arrSize; i++){
+        sum += arr[i];
+    }
+    return true;
+}
+
+int main() {
+    int arraySize = 0;
+    double sum = 0;
```
Comment (helen-black) [inline Lesson10/Lesson10_2.cpp]: = 0.0;

File: Lesson10/Lesson10_1.cpp (hunk 1 above)
Comment (vtx123) [reply Lesson10/Lesson10_1.cpp]: fixed

File: Lesson10/Lesson10_2.cpp (hunk 2 above)
Comment (vtx123) [reply Lesson10/Lesson10_2.cpp]: fixed

**Pull request merged**